Cirrious.FluentLayout/FluentLayout.cs

```diff
@@ -153,7 +154,7 @@ private void ThrowIfSecondItemAlreadySet()
 
         public IEnumerable<NSLayoutConstraint> ToLayoutConstraints()
         {
-            var constraint = NSLayoutConstraint.Create(
+            Constraint = NSLayoutConstraint.Create(
```

Comment:
For the first part, sorry I didn´t explain myself well enough. I´m not calling ToLayoutConstraints() myself. I´m just saying the whole library is useless without that method.

For the second part, I totally agree. This class should be reworked. 
What about `Constraint { get { return here the same as current ToLayoutConstraint } } `?